Manifest.toml

```diff
@@ -2350,9 +2355,9 @@ version = "0.6.0"
 
 [[deps.Rasters]]
 deps = ["Adapt", "ColorTypes", "CommonDataModel", "ConstructionBase", "Dates", "DimensionalData", "DiskArrays", "Extents", "FillArrays", "Flatten", "GeoFormatTypes", "GeoInterface", "Missings", "Mmap", "OffsetArrays", "ProgressMeter", "RecipesBase", "Reexport", "Setfield"]
-path = "../Rasters.jl"
+git-tree-sha1 = "1dceb16b3f6982f4f161d30b6afc96349ef86c38"
```

Comment:
This project now depends on a custom fork of Rasters.jl

https://github.com/open-AIMS/GBR-reef-guidance-assessment/tree/script-cleanup-2a?tab=readme-ov-file#setup